modules/math/shared/src/main/scala/gsp/math/skycalc/TwilightCalc.scala

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gsp.math.skycalc
+
+import cats.implicits._
+import gsp.math.JulianDate
+import gsp.math.Place
+import java.time.Instant
+import java.time.LocalDate
+
+trait TwilightCalc extends SunCalc {
+
+  /** Compute start and end of night for a particular date and place.
```

Comment:
Will it be in local time right?

Reply:
It takes a `LocalDate` as input and returns `Instant`s.